platform/src/main/resources/config/anchor-config-default-values.yaml

```
@@ -451,6 +451,8 @@ sep38:
   # Whether to enable SEP-38
   #
   enabled: false
+  # Whether to require SEP-10 authentication for SEP-38 requests.
+  requires_sep10: false
```

I feel `requires_sep10` is vague because some of the endpoints already requires sep10.

How about:
```suggestion
  enforce_sep10: false
```

`sep38.endpoints.info.authentication_required` seems a bit over engineering.